paper.css

```diff
@@ -9,19 +9,22 @@ body { margin: 0 }
 }
 
 /** Paper sizes **/
-body.A3           .sheet { width: 297mm; height: 419mm }
-body.A3.landscape .sheet { width: 420mm; height: 296mm }
-body.A4           .sheet { width: 210mm; height: 296mm }
-body.A4.landscape .sheet { width: 297mm; height: 209mm }
-body.A5           .sheet { width: 148mm; height: 209mm }
-body.A5.landscape .sheet { width: 210mm; height: 147mm }
+body.A3           .sheet { width: 297mm; height: 419mm;margin-left: auto; margin-right: auto; }
```

Comment:
@cognitom Thanks for replying, if I add these, papers will be displayed at middle of screen like:
![image](https://cloud.githubusercontent.com/assets/10969478/22403265/31fb017e-e64d-11e6-93c3-a6bf350e7858.png)
not on left side of screen, only looks good😁